llvm/lib/IR/Type.cpp

```diff
@@ -903,7 +947,7 @@ static TargetTypeInfo getTargetTypeInfo(const TargetExtType *Ty) {
 
   // DirectX resources
   if (Name.starts_with("dx."))
-    return TargetTypeInfo(PointerType::get(C, 0));
+    return TargetTypeInfo(PointerType::get(C, 0), TargetExtType::CanBeGlobal);
```

Comment:
@bogner @hekota I had to do this to keep `check-clang-codegenhlsl` passing. Does it look right?

Reply:
Yes, this looks reasonable to me.